cleanup.js

```diff
@@ -0,0 +1,34 @@
+const core = require('@actions/core');
+const { exec } = require("child_process");
+const path = require('path');
+const fb_env = {
+    'FIREBOLT_USER': core.getInput('firebolt-username'),
+    'FIREBOLT_PASSWORD': core.getInput('firebolt-password'),
+    'FIREBOLT_SERVER': core.getInput('api-endpoint'),
+    'FIREBOLT_DEFAULT_REGION': core.getInput('region')	     
+}
+
+
+function resolve_local_file(file_path) {
+        return path.join(__dirname, file_path)
+}
+
+function stop_all(db_name, on_success, on_error) {
+    python_bin = path.join(core.getState('python_path'), 'python');
```

Comment:
Does python here alias python3?

Reply:
Yes, because it's a binary from a virtual environment